app/src/ui/lib/sandboxed-markdown.tsx

```diff
@@ -161,11 +204,16 @@ export class SandboxedMarkdown extends React.PureComponent<
 
   public render() {
     return (
-      <iframe
-        className="sandboxed-markdown-component"
-        sandbox=""
-        ref={this.onFrameRef}
-      />
+      <div
+        className="sandboxed-markdown-iframe-container"
+        ref={this.onFrameContainingDivRef}
```

Comment:
If you don't plan on doing anything when the reference is set, I think you can just pass the ref itself here:
```suggestion
        ref={this.frameContainingDivRef}
```

Reply:
I was excited to use this... but when I tried it said it needed to be type LegacyRef<HTMLDivElement | undefined> When I set it to that and I tried to use it, it didn't have the style could be type 'string'.. So evidently using the method does some type casting for me that I would like to keep.